src/modules/HomePage/HomePage.tsx

```diff
@@ -1,9 +1,53 @@
+/* eslint-disable max-len */
+import React from 'react';
+import { PhoneCard, PhoneItem } from '../shared/components/PhoneCard';
 import './HomePage.module.scss';
 
-import React from 'react';
+const phone: PhoneItem = {
+  id: 'apple-iphone-11-128gb-green',
+  name: 'Apple iPhone 11 128GB Green',
+  image: 'img/phones/apple-iphone-11/green/00.jpg',
+  price: 1100,
+  fullPrice: 1050,
+  year: 2023,
+
+  capacity: '128GB',
+
+  color: 'green',
+
+  screen: "6.1' IPS",
+
+  ram: '4GB',
+
+  description: [
+    {
+      title: 'And then there was Pro',
+      text: [
+        'A transformative triple-camera system that adds tons of capability without complexity.',
+        'An unprecedented leap in battery life. And a mind-blowing chip that doubles down on machine learning and pushes the boundaries of what a smartphone can do. Welcome to the first iPhone powerful enough to be called Pro.',
+      ],
+    },
+    {
+      title: 'Camera',
+      text: [
+        'Meet the first triple-camera system to combine cutting-edge technology with the legendary simplicity of iPhone. Capture up to four times more scene. Get beautiful images in drastically lower light. Shoot the highest-quality video in a smartphone — then edit with the same tools you love for photos. You’ve never shot with anything like it.',
+      ],
+    },
+    {
+      title:
+        'Shoot it. Flip it. Zoom it. Crop it. Cut it. Light it. Tweak it. Love it.',
+      text: [
+        'iPhone 11 Pro lets you capture videos that are beautifully true to life, with greater detail and smoother motion. Epic processing power means it can shoot 4K video with extended dynamic range and cinematic video stabilization — all at 60 fps. You get more creative control, too, with four times more scene and powerful new editing tools to play with.',
+      ],
+    },
+  ],
+};
```

Comment:
You can import an array of phones and then use the first
`import phone from '../../static/api/phones.json';`